react/__tests__/__snapshots__/Telemarketing.test.tsx.snap

```diff
@@ -3,35 +3,35 @@
 exports[`<Telemarketing /> component should match snapshot with client 1`] = `
```

Comment:
This changes in css styles were proposital?

Reply:
yep